docs/product-portal/applications/consuming-megazord-libraries.md

```diff
@@ -118,6 +146,11 @@ Simply add this JAR to your classpath when running tests, like so:
 ```groovy
 dependencies {
   testImplementation "org.mozilla.appservices:full-megazord-forUnitTests:X.Y.Z"
+  modules {
+    module('org.mozilla.appservices:full-megazord') {
+      replacedBy('org.mozilla.appservices:full-megazord-forUnitTests', 'prefer the forUnitTests megazord in test configuration')
+    }
+  }
```

Comment:
I've hit bugs in the past there the test configuration was including *both* variants of the module, and adding this `replacedBy` rule avoids it. Basically I've copied what Fenix actually does in practice.